Mathlib/Algebra/ModEq.lean

```diff
@@ -321,6 +321,10 @@ theorem int_cast_modEq_int_cast {a b z : ℤ} : a ≡ b [PMOD (z : α)] ↔ a 
   norm_cast
 #align add_comm_group.int_cast_modeq_int_cast AddCommGroup.int_cast_modEq_int_cast
 
+@[simp, norm_cast]
+lemma intCast_modEq_intCast' {a b : ℤ} {n : ℕ} : a ≡ b [PMOD (n : α)] ↔ a ≡ b [PMOD (n : ℤ)] := by
```

Comment:
Why `intCast`, not `int_cast`? The previous lemma uses `int_cast`.

Reply:
It's one operation, so should be one atom. Let me fix the previous lemma.

Reply:
OTOH, we have [Function.Involutive.list_map](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/List/Basic.html#Function.Involutive.list_map)

Reply:
I don't think we should block this PR on a naming decision that mathlib isn't consistent about.